lib/logstash/inputs/jdbc.rb

```diff
@@ -42,12 +28,22 @@
 # }
 # ----------------------------------
 #
-# ==== statement vs statement_filepath
+# ==== Configuring SQL statement
 # 
-# A supplied sql statement is required. This can be passed-in via a 
-# statement field in the form of a string, or read from a file. The plugin will 
-# only accept one of the options. It cannot read a statement from a file as well as
-# from the `statement` configuration parameter.
+# A sql statement is required for this input. This can be passed-in via a 
+# statement option in the form of a string, or read from a file (`statement_filepath`). File 
+# option is typically used when the SQL statement is large or cumbersome to supply in the config.
+# The file option only supports one SQL statement. The plugin will only accept one of the options.
+# It cannot read a statement from a file as well as from the `statement` configuration parameter.
+#
+# ==== Predefined Parameters
+#
+# Some parameters are built-in and can be used from within your queries.
+# Here is the list:
+#
+# |==========================================================
+# |sql_last_start |The time the last query executed in plugin
```

Comment:
yup. will clarify this. `sql_last_start` is set to the start of time before a query is executed